examples/llm_ptq/scripts/huggingface_example.sh

```
@@ -223,7 +251,21 @@ if [[ $TASKS =~ "quant" ]] || [[ ! -d "$SAVE_PATH" ]] || [[ ! $(ls -A $SAVE_PATH
     # (lm_eval/mmlu/simple_eval) deploy the checkpoint themselves, so it is redundant there.
     if [[ $TASKS =~ "quant" ]]; then
-        python run_tensorrt_llm.py --checkpoint_dir=$SAVE_PATH $RUN_ARGS
+        if $VLM; then
+            # VLMs use the TRT-LLM multimodal quickstart for the deploy smoke test.
+            if [ -z "$TRT_LLM_CODE_PATH" ]; then
+                TRT_LLM_CODE_PATH=/app/tensorrt_llm # default path for the TRT-LLM release docker image
+                echo "Setting default TRT_LLM_CODE_PATH to $TRT_LLM_CODE_PATH."
+            fi
+            QUICK_START_MULTIMODAL=$TRT_LLM_CODE_PATH/examples/llm-api/quickstart_multimodal.py
+            if [ -f "$QUICK_START_MULTIMODAL" ]; then
+                python3 $QUICK_START_MULTIMODAL --model_dir $SAVE_PATH --modality image
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)